engine/src/deploy.cpp

```diff
@@ -398,21 +398,67 @@ static bool MCDeployWriteDefineLicenseSection(const MCDeployParameters& p_params
 			t_edition = 1;
 			break;
 		
-		case kMCLicenseClassEvaluation:
```

Comment:
I know I wrote this code - however, I don't know why I didn't just use the values from the kMCLicenseClass enum... Can you see a reason why we can't just use (uint1)MClicenseparameter.license_class; and set it directly back in the capsule load?

Reply:
what type are enums? are they consistent for all our toolchains?

Reply:
Maybe not, but that's why I suggested casting... Something like...
```
IO_write_uint1(..., (uint1)MClicenseparameter.license_class);

...

uint1 t_class;
IO_read_uint1(..., &t_class);
MClicenseparameters.license_class = (MCLicenseClass)t_class
```

Reply:
ok, one less thing to update if we add more editions!

Reply:
OK, done... although by the look of the tests I've broken standalone building here.

Reply:
ah, found it I think